src/registry.ts

```diff
@@ -297,7 +297,7 @@ export function getEntryUrlForPortal(
   const query = {
     publickey: ensurePrefix(publicKey, ED25519_PREFIX),
     datakey: dataKeyHashHex,
-    timeout: DEFAULT_GET_ENTRY_TIMEOUT,
+    timeout: DEFAULT_GET_ENTRY_TIMEOUT.toString(),
```

Comment:
```suggestion
    timeout: DEFAULT_GET_ENTRY_TIMEOUT,
```

`makeUrl` should be happy to accept numbers